.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules
 /coverage
 .DS_Store
+*.log
+.vscode/
```

Comment:
`.vscode` should be in the global `.gitignore`.. Dont know if its a good thing to put it here.